lib/main_common.pm

```diff
@@ -2801,9 +2801,23 @@ sub load_publiccloud_tests {
         loadtest "publiccloud/ssh_interactive_start", run_args => $args;
         loadtest "publiccloud/instance_overview";
         load_extra_tests_prepare();
+        load_extra_tests_docker();
         load_extra_tests_console();
```

Comment:
`elsif (get_var('PUBLIC_CLOUD_CONTAINERS')) {` branch fully repeat `PUBLIC_CLOUD_QAM`  one except this one line . so maybe we can just do this ? : 
```suggestion
        load_extra_tests_console() unless(get_var('PUBLIC_CLOUD_CONTAINERS'));
```

Reply:
Agree. And vice versa

Reply:
Nice idea! I included it in the PR